package.json

```diff
@@ -73,41 +73,41 @@
 	},
 	"devDependencies": {
 		"@ava/typescript": "^1.1.1",
-		"@rollup/plugin-typescript": "^5.0.1",
+		"@rollup/plugin-typescript": "^5.0.2",
 		"@sindresorhus/tsconfig": "^0.7.0",
-		"@types/chrome": "0.0.117",
-		"@types/codemirror": "0.0.96",
+		"@types/chrome": "0.0.122",
+		"@types/codemirror": "0.0.97",
 		"@types/copy-webpack-plugin": "^6.0.0",
-		"@types/firefox-webext-browser": "^78.0.0",
+		"@types/firefox-webext-browser": "^78.0.1",
 		"@types/jsdom": "^16.2.3",
 		"@types/mini-css-extract-plugin": "^0.9.1",
-		"@types/react": "^16.9.41",
+		"@types/react": "^16.9.45",
 		"@types/terser-webpack-plugin": "^3.0.0",
-		"ava": "^3.10.0",
+		"ava": "^3.11.1",
 		"chrome-webstore-upload-cli": "^1.2.0",
 		"copy-webpack-plugin": "^6.0.3",
-		"css-loader": "^3.6.0",
+		"css-loader": "^4.2.1",
 		"daily-version": "^2.0.0",
 		"dot-json": "^1.2.0",
 		"eslint-config-xo-react": "^0.23.0",
-		"eslint-plugin-react": "^7.20.3",
-		"eslint-plugin-react-hooks": "^4.0.5",
-		"jsdom": "^16.2.2",
+		"eslint-plugin-react": "^7.20.5",
+		"eslint-plugin-react-hooks": "^4.0.8",
+		"jsdom": "^16.4.0",
 		"mini-css-extract-plugin": "^0.9.0",
 		"npm-run-all": "^4.1.5",
-		"rollup": "^2.19.0",
+		"rollup": "^2.23.1",
 		"size-plugin": "^2.0.1",
 		"string-replace-loader": "^2.3.0",
 		"stylelint": "^13.6.1",
 		"stylelint-config-xo": "^0.19.0",
 		"terser-webpack-plugin": "^3.0.6",
```

Comment:
I'm not updating this one yet because in the next version (https://github.com/webpack-contrib/terser-webpack-plugin/commit/100e38e0db035f6eb21bbec2bbc1f8b12aee9f60) they start affecting `copy-webpack-plugin` files too, which means that our neat [webextensions.min.js](https://www.unpkg.com/browse/webextension-polyfill@0.6.0/dist/browser-polyfill.min.js) is stripped of license information and unnecessarily "beautified"

I initially extracted this file because:

- I want `browser` to be a global
- I thought AMO [preferred](https://extensionworkshop.com/documentation/publish/third-party-library-usage/) unmodified libraries
- It's minified by default!
- It can be used by all entry points without being bundled in each one of them

There are 2 possible solutions to this:

- find/add a way for `terser-webpack-plugin` to exclude this file
- have webpack create a vendor file

I tried the second one in https://github.com/sindresorhus/refined-github/pull/3157 already but that failed

Reply:
May be possible with https://github.com/webpack-contrib/terser-webpack-plugin#exclude

Reply:
Fixed!